Formula/f/fernflower.rb

```diff
@@ -0,0 +1,42 @@
+class Fernflower < Formula
+  desc "Advanced decompiler for Java bytecode"
+  homepage "https://github.com/JetBrains/fernflower"
+  url "https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/java/java-decompiler-engine/252.25557.131/java-decompiler-engine-252.25557.131.jar"
```

Comment:
looks like the jar is the same as `https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/java/java-decompiler-engine/252.26199.169/java-decompiler-engine-252.26199.169.jar` (at least based on sha)

Reply:
because there were no changes to the decompiler between these releases ([link](https://github.com/JetBrains/intellij-community/compare/idea/252.26199.169...idea/252.25557.131)).

[These releases](https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/java/java-decompiler-engine/maven-metadata.xml) correspond to [stable IntelliJ IDEA releases](https://www.jetbrains.com/idea/download/other.html). It's possible that the decompiler sources do change between them.

Reply:
so it would always have a new release even though there is no source code changes?

Reply:
It's possible, yes. But I'm not sure if it's wrong.
Fernflower doesn't have its own  versioning. It's versioned together with IntelliJ releases.